### ISSUE TITLE
Remove inspection from sighted events 

### ISSUE THREAD

**Issue by Samuel-Baird** [thewca/wca-regulations#863]
Edit: named this a bit hastily, this is just a pros and cons list of removing inspection from speed solving events. 

Pros:
*This makes it very simple to explain to the judging process to new judges. With the current system the judge has to look at the stop watch and competitor simultaneously, remember to call out 8 seconds, 12 seconds, take note of whether they pass 15 seconds, take note if they pass 17 seconds. Removing inspection allows the competitors to focus entirely on the competitor which could even prevent cheating by means such as applying moves during inspection. 
*stopwatches and judges may not be precise. A judge may start the stop watch slightly before or after lifting the cover, this could result in someone with 15.1 seconds of inspection not receiving a penalty while a competitor who inspected for 14.9 seconds may revive a penalty. By removing inspection we create more fair and equal conditions amongst competitors. 
*It makes speed solving events more valid. In the eyes of many people it would make sense to start the timer as soon as you start seeing the cube. It may not truly take a competitor 10 seconds to solve a cube, rather it takes them 12 seconds of looking at it and planning moves and then 10 seconds of turning it to solve. 
*This makes it very simple to explain the competing process for new competitors. Instead of needing to start the timer within the certain time limit it would instead be like blindfolded where the competitor removes the cube cover when they are ready. A much simpler process than what we currently have. I personally have judged several new competitors who have gone over 17 seconds waiting for me to indicate that they should start, the new system would have no similar issues. 
*it adds a new element to speed solving, new strategies and methods would likely be developed to achieve good results with the new system. 

Cons:
*this will result in slower solves, we would likely need to treat this similar to mbld and mbld old style. Our current results would archived as “old style” and our “new” events would start with their current names and no results. 
Con: some competitors heavily rely on inspection, their results would suffer compared to other competitors who use little inspection.

**Comment by maconard** [comment]
10/10 shitpost, I guess?

**Comment by Sixstringcal** [comment]
> 10/10 shitpost, I guess?

In comparison to some of the other ideas on here (even some suggested by some WRC members) this is actually pretty reasonable imo.  I'm not for it in the slightest, but it's just as valid of a point on here as some other content on here.

**Comment by aneurinhunt** [comment]
This is pretty much the biggest change you can make to the regulations.

**Comment by dancing-jules** [comment]
> treat this similar to mbld and mbld old style

I don't find it comparable at all. MBLD had very few competitors back then (and most of them had a result with 2 attempted cubes) and it was only one event. 

Doing this for all sighted events in 2020 would be akin to scraping the whole database and affecting 14 events and 130k+ people.

**Comment by andrewtyberg** [comment]
First of all, there is no way this would happen in 2020. Seeing that the removal of feet was announced over a year in advance and is still extremely controversial, this would need to be announced AT LEAST that far in advance. If this were to actually happen, I would see it as intense discussion and debate over the next several months, trying it out as unofficial events at competitions, and then a final announcement of its removal long before it happens. The soonest I could see this actually happening would be following worlds 2021.

This would be such a monumental change to the WCA and the way people solve puzzles. I think, in theory, it is more logical than having inspection. In other words, if we were to start over with creating the WCA from scratch, we would choose no inspection. But we've been doing inspection for so long, and removing it would essentially invalidate all previous results in most events. From then on, all results in 12 or 13 events (depending on if feet stays or not) would be recorded differently, all PRs, NRs, CRs, and WRs, and really ALL RESULTS would have to be reset.

In summary, I think this would have been a good thing back when the WCA was starting, but with how used we've gotten to inspection, I think it would be too drastic of a change to make at this point.

**Comment by xsrvmy** [comment]
Even worse this would probably remove certain more inspection reliant methods (roux and zz?)

**Comment by Samuel-Baird** [comment]
Closing for now. This change is a bit more extreme and would only happen if the WCA was doing some large scale remodeling.
